content/en/includes/federation-deprecation-warning-note.md

```diff
@@ -0,0 +1,3 @@
+`Federation V1`, the Kubernetes federation API which reuses the Kubernetes API resources 'as is' never achieved GA status and is no longer under active development. It is documented here for historical purposes only, and its use is not recommended.. 
+
+However, there is a `Federation V2` effort which is not GA but is the intended replacement, more details on [Kubernetes Federation v2 git repository](https://github.com/kubernetes-sigs/federation-v2).
```

Comment:
Replace with:
```
For more information, see ["Kubernetes Federation v2"](https://github.com/kubernetes-sigs/federation-v2).
```

Reply:
done @zacharysarah , please help review again

Reply:
I think it did make sense to point out the fact (which probably is important to be pointed out to the readers), as written by @DanyC97  -> `but is the intended replacement`.
@zacharysarah @DanyC97 english is not my first language, would it be possible for you to frame a sentence which ensures that this fact is also included.